rules/microsoft_rules/microsoft_graph_passthrough.py

```diff
@@ -14,6 +14,8 @@ def dedup(event):
 
 
 def severity(event):
+    if event.get("severity") == "informational":
```

Comment:
Want to `lower` this string?

Reply:
eh, may as well 👍 . in a data sample, all values were already lowercased